frontend/front/src/pages/Public/Pages/BenefitsHeatPumps.js

```diff
@@ -19,44 +20,45 @@ import heatIcon from "../../../assets/images/Icons/heat.png";
-    bodyLink: "",
-    link: "https://goclean.masscec.com/clean-energy-solutions/air-source-heat-pumps",
-  },
-  {
-    icon: communityIcon,
-    title: "Stronger Communities",
-    body: "Boston residents want to stay in their homes and keep their communities strong, even while facing challenges from rising costs. Switching to heat pumps can lower utility costs, reducing the financial challenge of staying in the community. By also adding active cooling capacity, ASHPs also improve the “housing resiliency” of entire neighborhoods as annual temperatures rise. With both energy bills and temperatures on the rise, housing resiliency is more important than ever to empower communities to stay strong and stay together. To learn more, ",
-    bodyLink: "",
-    link: "https://www.energy.gov/policy/articles/heat-pumps-keep-homes-warm-and-bills-low-winter",
-  },
-  {
-    icon: heatIcon,
-    title: "Lower Carbon Emissions",
-    body: "Heat pumps are highly efficient heating and cooling systems that are electrically-powered. As such, they become “cleaner” whenever the source of their electric power becomes cleaner. Gas or oil fueled heating/cooling cannot benefit in the same way. How much of a difference will switching to ASHPs make in your case? Find specifics on the climate impact of a switch to heat pumps, ",
-    bodyLink: "",
-    link: "https://goclean.masscec.com/clean-energy-solutions/",
-  },
-];
-
 const BenefitsHeatPumps = () => {
   const theme = useTheme();
   const isSmallerThanSm = useMediaQuery(theme.breakpoints.down("sm"));
 
+  const { t } = useTranslation();
+
+  const cardBenefits = [
+    {
+      icon: moneyIcon,
+      title: t("public.benefits-heat-pump.cardBenefits.item1.title"),
+      body: t("public.benefits-heat-pump.cardBenefits.item1.body1"),
+      bodyLink:
+        "https://www.amerenmissourisavings.com/hvac-air-source-heat-pump-education-flyer/",
+      body2: `). ${t("public.benefits-heat-pump.cardBenefits.item1.body2")}`,
+      link: "https://www.masssave.com/residential/heating-comparison-calculator",
+    },
+    {
+      icon: snowSunIcon,
+      title: t("public.benefits-heat-pump.cardBenefits.item2.title"),
+      body: t("public.benefits-heat-pump.cardBenefits.item2.body1"),
+      bodyLink: "",
+      link: "https://goclean.masscec.com/clean-energy-solutions/air-source-heat-pumps",
+    },
+    {
+      icon: communityIcon,
+      title: t("public.benefits-heat-pump.cardBenefits.item3.title"),
+      body: t("public.benefits-heat-pump.cardBenefits.item3.body1"),
+      bodyLink: "",
+      link: "https://www.energy.gov/policy/articles/heat-pumps-keep-homes-warm-and-bills-low-winter",
+    },
+    {
+      icon: heatIcon,
+      title: t("public.benefits-heat-pump.cardBenefits.item4.title"),
+      body: t("public.benefits-heat-pump.cardBenefits.item4.body1"),
+      bodyLink: "",
+      link: "https://goclean.masscec.com/clean-energy-solutions/",
+    },
+  ];
```

Comment:
Check out how the "click here" links appear.
All the English ones appears like this:
> To compare your system and to see how much you could save. [click here](https://www.masssave.com/residential/heating-comparison-calculator).

It should have a comma:
> To compare your system and to see how much you could save, [click here](https://www.masssave.com/residential/heating-comparison-calculator).

For Creole and Spanish, in the Stronger Communities section, the Click Here link appears like this:
>Aprender más.[haga clic aquí](https://www.energy.gov/policy/articles/heat-pumps-keep-homes-warm-and-bills-low-winter).

Again should be ", aprender más" (with lowercase aprender).

Reply:
Fixed it: https://github.com/codeforboston/urban-league-heat-pump-accelerator/pull/489/commits/4a8a9dde9fce09044cb4e2e39a4c4a90cb87888f